pkg/proxy/ipvs/proxier.go

```diff
@@ -300,6 +301,11 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 	return ips, nil
 }
 
+// BindedIPs returns all addresses there were binded to the IPVS dummy interface kube-ipvs0
```

Comment:
"there were" => "that are" ?
(probably my mistake in the first place)

Reply:
fixed